src/main/java/org/jenkinsci/plugins/ghprb/GhprbCause.java

```diff
@@ -53,7 +53,12 @@ public String getCommit() {
 	}
 	
 	public boolean isMerged() {
-		return merged;
+        // hardcode hack: never automatically merge a commit for a pull request build.
+        // see various bugs that are affected by this. Header descriptions for edx are in
+        // https://openedx.atlassian.net/browse/TE-703
+
+        //return merged;
+        return false;
```

Comment:
I considered that, but wanted to leave it down here with the idea that it eventually gets fixed in this method. Plus I wanted to conserve where the attribute was coming from (the API call).